mailing_list/migrations/0013_auto_20160718_1945.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mailing_list', '0012_supersender'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CourseSettings',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('canvas_course_id', models.IntegerField()),
```

Comment:
Sure. I think it's fine to leave it the way it is and investigate a bit for future knowledge as we have time during ACP etc.